tensorflow/compiler/tf2tensorrt/convert/convert_nodes.cc

```diff
@@ -1623,7 +1655,7 @@ tensorflow::Status ConvertConv2DHelper(OpConverterParams* params, int group) {
   const nvinfer1::DimsHW stride(tf_stride[h_index], tf_stride[w_index]);
   if (params->validation_only) return tensorflow::Status::OK();
 
-  const nvinfer1::ITensor* tensor = inputs.at(0).tensor();
+  const nvinfer1::ITensor* tensor = input.tensor();
```

Comment:
nit: This is the only place where we seem to be using `input` variable. If that is the case, then we can directly extract tensor at line 1596.